src/test/java/uk/gov/hmcts/divorce/citizen/event/CitizenSwitchedToSoleTest.java

```diff
@@ -119,6 +119,34 @@ void givenEventStartedWithValidJointCaseForApplicant1SwitchToSoleWithApplicant2N
 
         final AboutToStartOrSubmitResponse<CaseData, State> response = citizenSwitchedToSole.aboutToSubmit(caseDetails, caseDetailsBefore);
 
+        verify(notificationDispatcher).send(applicant1SwitchToSoleNotification, caseData, caseDetails.getId());
+        verifyNoMoreInteractions(notificationDispatcher);
+        verify(ccdAccessService, times(0)).unlinkApplicant2FromCase(anyLong(), anyString());
```

Comment:
could use verifyNoInteractions instead of verify(ccdAccessService, times(0))

Reply:
ccdAccessService is called before unless you can do verifyNoInteractions(ccdAccessService.unlinkApplicant2FromCase)

Reply:
actually ignore this I did not spot the other usage